HospitalWebApp/Controllers/FeedbackController.cs

```diff
@@ -22,79 +22,73 @@ public FeedbackController(UserFeedbackService userFeedbackService)
         }
 
         /// <summary>
-        /// Updates the given <see cref="UserFeedback"/>
+        ///     Creates a new <see cref="UserFeedback"/> from the given <see cref="UserFeedbackDto"/> object. 
         /// </summary>
-        /// <param name="userFeedback"></param>
-        /// <returns>An <see cref="IActionResult"/> representing a result of the operation.</returns>
-        [HttpPut]
-        public IActionResult Update(UserFeedbackDto userFeedbackDto)
-        {
-            try
-            {
-                UserFeedbackValidator.validate(userFeedbackDto);
-                var userFeedback = UserFeedbackAdapter.userFeedbackDtoToUserFeedback(userFeedbackDto);
-                return Ok(_userFeedbackService.Update(userFeedback));
-            }
-            catch(ReferenceConstraintException e)
-            {
-                return BadRequest(e.Message);
-            }
-            catch(ValidationException e)
-            {
-                return BadRequest(e.Message);
-            }
-        }
-        
-        /// <summary>
-        ///  Creates new user feedback.
-        /// </summary>
-        /// <param name="userFeedback"></param>
-        /// <returns></returns>
```

Comment:
Update method currently voted out by the Web team as redundant. If the need arises in the following sprints, it will be reinstated.